src/platform/nrfconnect/BUILD.gn

```diff
@@ -77,4 +78,11 @@ static_library("nrfconnect") {
       "OTAImageProcessorImpl.h",
     ]
   }
+
+  if (chip_malloc_sys_heap) {
+    sources += [
+      "../Zephyr/SysHeapMalloc.cpp",
```

Comment:
Shouldn't we add this files to the Telink build as well? In other case invoking methods in generic Zephyr code may fail for their platform.

Reply:
I simply didn't want to do everything for them. I only made sure that I didn't break anything on their platform, and that they can easily adapt to the CHIP_MALLOC_SYS_HEAP, but I'm not sure they are really interested in using it.